src/h3lib/lib/faceijk.c

```diff
@@ -415,7 +415,7 @@ void _geoToHex2d(const GeoCoord* g, int res, int* face, Vec2d* v) {
                       _posAngleRads(_geoAzimuthRads(&faceCenterGeo[*face], g)));
 
     // adjust theta for Class III (odd resolutions)
-    if (isResClassIII(res)) theta = _posAngleRads(theta - M_AP7_ROT_RADS);
+    if (isResDigitClassIII(res)) theta = _posAngleRads(theta - M_AP7_ROT_RADS);
```

Comment:
I'm finding this one a little confusing, naming-wise. What about going the other way, and using `isClassIII` for testing the resolution? or, alternatively, using a `IS_CLASS_III` macro?

Reply:
So `isResClassIII` was taken over by `h3IsResIII` (or something like that) and I had to rename this function. I didn't want to refactor it into a macro or anything because it would make proving that the renaming was correct even more difficult than it already is in a code review. Having this renaming PR as straightforward as possible makes reviewing the monstrosity more manageable.

Reply:
Ok, stet

Reply:
I would suggest renaming to `isResolutionClassIII` or similar, there is no use of indexing digits ehre. I understand the concern with renaming to `isResClassIII`, which would be my preferred name.